docs/practices/pull_requests.md

```diff
@@ -0,0 +1,79 @@
+  If it's straightforward, go ahead and merge it.
+- **Straightforward and non-controversial PRs** with at least one approval from a {team}`Maintainer Team` member and no objections may be merged by anybody, anytime.
+  If you approve the PR, go ahead and merge it!
+  If you don't immediately merge it, provide a rationale for why not.
+  If somebody else approves your PR and you think it's safe to merge, feel free to self-merge.
+- **Complex or potentially controversial PRs** may be merged with at least one approval from a {team}`Maintainer Team` member, no objections from any team member, and after has been open for around five working days.
+  Use your best judgment for how long to wait to ensure inclusive decision making process and be sure to [include others in discussion](prs:principles).
+  The more complex, the more time you should give people.
+  The more people agree, the more quickly you can safely merge.
+- **Changes to team policies** may be merged with at least two approvals from {team}`JupyterHub Council` members, no objections from any team member, and has been open for around five working days (follow the same guidelines as above).
+- **Proposals without consensus** may be merged if the {team}`JupyterHub Council` has majority agreement to merge, and if there have been at least five working days to discuss.
+
+## What to look for in a review
+
+As a team member, you're encouraged to help others contribute to the project
+by reviewing their code, guiding them towards making a contribution and
+improving it, and ultimately merging their contribution into the project.
+
+Having merge rights is both a privilege and a responsibility - please be
+thoughtful when using it! To that extent, here are a few guidelines when
+deciding to merge things into one of our repositories:
+
+- **It's quality code**. We know this is somewhat subjective, but
+  ensure that the code is well-organized and thoughtfully-written, that any
+  new features are documented, and that it abides by best-practices in Python,
+  JavaScript, etc.
+  Do not nit-pick stylistic preferences, but if you genuinely think there's a clear way to improve the code, please make suggestions.
+- **There are tests and they pass**. We try not to merge any new features (or
+  bugfixes!) without adding tests for them. It's easy to consider something
+  minor enough that it doesn't warrant a test, but try to avoid doing this!
+  Adding tests usually only takes a moment, and our future selves will thank
+  us for it later.
+- **There is documentation**. If we're changing or adding functionality, ensure that there is documentation describing the change to others.
+  Functionality that isn't documented is much less discoverable and maintainable.
+- **The code is secure**. We must avoid introducing changes that may lead to a security vulnerability. If your code is complicated please include comments to ensure future maintainers do not inadvertently introduce a new vulnerability.
```

Comment:
I'm not sure, but from this diff view it looks there's a bit of inconsistency in this section around splitting by sentence and splitting by line length?

```suggestion
- **The code is secure**. We must avoid introducing changes that may lead to a security vulnerability.
  If your code is complicated please include comments to ensure future maintainers do not inadvertently introduce a new vulnerability.
```